systemtest/src/main/resources/olm/subscription.yaml

```diff
@@ -20,4 +20,5 @@ spec:
         value: "${STRIMZI_FULL_RECONCILIATION_INTERVAL_MS}"
       - name: STRIMZI_OPERATION_TIMEOUT_MS
         value: "${STRIMZI_OPERATION_TIMEOUT_MS}"
-
+      - name: STRIMZI_PERMISSIONS_MODE
+        value: "${STRIMZI_PERMISSIONS_MODE}"
```

Comment:
How does this work within the OLM deployment where the RBAC files are hard coded? I would expect that it makes no sense here.